tests/integration/components/project-settings/general-settings/device-preferences-automated-dast-test.js

```diff
@@ -0,0 +1,183 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupIntl } from 'ember-intl/test-support';
+import { render, click, findAll } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { selectChoose } from 'ember-power-select/test-support';
+
+import styles from 'irene/components/ak-select/index.scss';
+import ENUMS from 'irene/enums';
+
+const classes = {
+  dropdown: styles['ak-select-dropdown'],
+  trigger: styles['ak-select-trigger'],
+  triggerError: styles['ak-select-trigger-error'],
+};
+
+module(
+  'Integration | Component | project-settings/general-settings/device-preferences-automated-dast',
+  function (hooks) {
+    setupRenderingTest(hooks);
+    setupMirage(hooks);
+    setupIntl(hooks);
+
+    hooks.beforeEach(async function () {
+      this.server.createList('organization', 1);
```

Comment:
Arrange `beforeEach` hook properly. Section models, server mocks and property setters appropriately.